src/hocon_schema.erl

```diff
@@ -300,7 +314,7 @@ find_error(Proplist) ->
     case do_find_error(Proplist, []) of
         [{validation_error, _, _, _} | _] = Errors ->
```

Comment:
not as a request to change (at least not in this PR),
it's better to run the errors into `map()`:s